Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidationTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using PopValidations.Validations;
+using PopValidations_Tests.ValidationsTests.GenericTestableObjects;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace PopValidations_Tests.ValidationsTests;
+
+public class IsNotEmptyValidationTests

```

Comment:
You should be getting namespace conflicts., Im guessing you did, and deleted the namespce.
If you checked the other files, you'd have seen the solution to this.
There should be an _  between Validation and Tests. in this class/file name.
And the namespace needs to be updated to the folder name.

Reply:
My bad, I hadn't fetched the repo in far too long it seems.
I only fetched the latest version just before submitting the pull request.
It said there weren't any merge conflicts and I didn't consider that there'd be so many changes.
I should have run the tests again... my bad.